tests/test_actions.py

```diff
@@ -684,6 +684,28 @@ def test_list_change_sets_sends_correct_request(self):
             kwargs={"StackName": sentinel.external_name}
         )
 
+    @patch("sceptre.plan.actions.urllib.parse.urlencode")
+    @patch("sceptre.plan.actions.ConnectionManager.connection_manager.call")
+    def test_list_change_sets_url_mode(
```

Comment:
can we get a test that verifies multiple change sets and another one that verifies no change sets?

Reply:
done